spec/rdkafka/admin_spec.rb

```diff
@@ -14,6 +14,8 @@
   let(:topic_name)               { "test-topic-#{Random.new.rand(0..1_000_000)}" }
   let(:topic_partition_count)    { 3 }
   let(:topic_replication_factor) { 1 }
+  let(:topic_config)             { {"cleanup.policy" => "compact", "min.cleanable.dirty.ratio" => 0.8} }
+  let(:invalid_topic_config)     { {"cleeeeenup.policee" => "campact"} }
```

Comment:
I think most spellcheck would have no suggestions for these misspellings!  😆

Reply:
Haha, yeah perhaps it's a little over the top 😄